backend/features/sample_scenario.feature

```diff
@@ -1,6 +1,27 @@
 Feature: ToDo Lists
   Sample scenario for QA TopAcademy Cucumber Evaluation
-
-  Scenario: Open app page
+  Background:
     Given I am in ToDo Lists app
-    Then I should see a button with text "Add"
+
+  Scenario Outline: Add some lists to the to-do list
+    When I add a "<list>" to the to-do list
+    Then I should see the "<list>" added to the list
+    And I should see the number of lists is not equal to 0
+
+    Examples:
+      |list|
```

Comment:
JFYI, there is a [shortcut](https://www.jetbrains.com/help/ruby/reformat-and-rearrange-code.html#reformat_code) to reformat the code in Rubymine. It helps to keep tables and their borders aligned

Reply:
it is done